tools/packgen/src/PackGen.cpp

```diff
@@ -164,6 +168,10 @@ bool PackGen::ParseManifest(void) {
 
     // Pack
     YAML::Node packs = manifest["packs"];
+    if (!packs.IsDefined()) {
+      cerr << "packgen error: mandatory 'packs' section is not defined!" << endl;
```

Comment:
I am not familiar with yaml and therefore unsure whether 'packs' is a key or 'section'?